packages/common/lib/typescript.d.ts

```diff
@@ -3332,7 +3332,7 @@ declare namespace ts {
             isNonTsProject(): boolean;
             isJsOnlyProject(): boolean;
             static resolveModule(moduleName: string, initialDir: string, host: ServerHost, log: (message: string) => void): {} | undefined;
-            readonly jsDocParsingMode: JSDocParsingMode | undefined;
+            readonly jsDocParsingMode: JSDocParsingMode;
```

Comment:
This doesn't seem to fix the issue when I try it out. I don't understand why TypeScript started failing with the original error.

Reply:
Oh, sorry I went from the other issue straight here and didn't notice the number changed.

The only thing about making this change is that these are the declaration files that typescript distributes and also it seems like this can be undefined. Does this error occur in TypeScript 5.4's declaration file? Maybe it will just be fixed once ts-morph is upgraded to that? (I'm slowly working on this)

Reply:
Yeah, I think this was fixed by the upgrade to TS 5.5. The declaration is now:

```
readonly jsDocParsingMode?: JSDocParsingMode | undefined;
```

Reply:
That makes sense, in that case I'll close this and wait for the upgrade to TS 5.4. Is there an issue or PR I can track for that? Thanks!

Reply:
It's released as of today.